examples/arc.py

```diff
@@ -0,0 +1,59 @@
+#! /usr/bin/env python3
+#
+# Test for a push-button monome arc
+
+import asyncio
+import monome
+
+class ExampleArc(monome.Arc):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+
+class ExampleArcApp(monome.ArcApp):   
+    def __init__(self):
+        arc = ExampleArc()
+        self.pos = [0, 0, 0, 0];
+        super().__init__(arc)
+        
+    def on_arc_ready(self):
+        print('Ready, clearing all rings...')
+        for n in range(0,4):
+            self.arc.ring_all(n,0)
+            
+    def on_arc_disconnect(self):
+        print('Arc disconnected.')
+        
+    def on_arc_delta(self, ring, delta):
+        print('Ring: {} Delta: {}'.format(ring, delta))
+        
+        oldPos = self.pos[ring]
+        newPos = oldPos + delta
```

Comment:
pymonome code and the examples use snake_case, please consider changing camelCase identifiers to snake_case, e.g. new_pos/old_pos